docs/deployment/connect-scm.md

```diff
@@ -0,0 +1,91 @@
+---
+slug: connect-scm 
+title: Connect a source code manager
+hide_title: true
+description: Connect a GitHub or GitLab organization to manage user authentication.
+tags:
+  - Semgrep Cloud Platform
+  - Team & Enterprise Tier
+  - Deployment
+---
+
+# Connect a source code manager
+
+:::note Your deployment journey
+- You have [created a Semgrep account and organization](/deployment/create-account-and-orgs). 
+:::
+
+Linking a source code manager provides the following benefits:
+
+- Allows the Semgrep org membership to be managed by GitHub or GitLab.
+- For GitHub users:
+    - Provides Semgrep access to post PR or MR comments.
+    - For GitHub Actions users: Enables you to add a Semgrep CI job to repositories in bulk.
+
+You can only connect your Semgrep organization to the source code manager that you originally logged in with. If your organization uses both GitHub and GitLab to manage source code, log in with the source code manager that you would prefer to use to manage Semgrep org membership. You can still scan repositories from other sources.
+
+The process to connect a source code manager depends on whether your SCM tool is cloud-hosted by the service provider, hosted on-premise, or hosted as a single-tenant by the service provider.
+
+:::note Review necessary permissions 
```

Comment:
Does it make sense to combine this admonition with the first one on the page? Then people could combine the preliminary work they need to do all at once

Reply:
I'll include the pre-deployment checklist as the first bullet in **Your deployment journey**.